packages/flutter/lib/src/widgets/focus_scope.dart

```diff
@@ -433,7 +433,11 @@ class _FocusState extends State<Focus> {
     _focusAttachment.reparent();
     return _FocusMarker(
       node: focusNode,
-      child: widget.child,
+      child: Semantics(
+        focusable: focusNode.canRequestFocus,
```

Comment:
Where is focusable added to the Semantics widget? Is this PR missing some files?